microsoft-edge/web-platform/user-agent-guidance.md

```diff
@@ -26,15 +28,15 @@ This article describes the methods Microsoft Edge supports for retrieving user a
 
 ## Feature detection
 
-Microsoft recommends detecting if a feature is supported in your browser. We do not recommend that you use feature detection to detect a specific browser. For more information see [feature detection][MdnLearnToolsTestingCrossBrowserTestingFeatureDetection].
+Microsoft recommends detecting if a feature is supported in your browser. For more information, see [feature detection][MdnLearnToolsTestingCrossBrowserTestingFeatureDetection].
 
-If you must detect browsers, we recommend using client hints as follows.
+If you must detect browsers, Microsoft recommends using client hints as follows.
 
-## User Agent Client Hints
+## User-Agent Client Hints
 
 Microsoft Edge supports User-Agent Client Hints starting with version 90.
 
-User-Agent Client Hints easily retrieve basic data such as the browser name and whether or not it is mobile. Also, hints are more likely to be supported in the long term as User-Agent string usage declines.
+User-Agent Client Hints is a cleaner, more privacy-preserving way to access browser information such as browser name, version number, platform, and more. Soon the User-Agent string will be frozen and deprecated by various browsers. For example, the Chrome Platform Status site describes the change in [Feature: Reduce User-Agent string information][https://www.chromestatus.com/feature/5704553745874944].

```

Comment:
Looks like the link at the end of the line wasn't quite formatted correctly. When linking to a URL, I think you need parentheses:

Ex: `[Google](https://google.com)` -> [Google](https://google.com)

Reply:
The "review" environment doesn't some resolve links ... and I'm not sure why we use [][] instead of []() ... but both work. Maybe ask Reeza  B^)

Reply:
Tip for everyone:  When you're reading docs at a "review" url, eg
https://review.docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/whats-new/2021/07/devtools?branch=user%2Fv-mhoffman%2Fwhatsnew93
and you click a link and get a 404, simply delete "review." from the URL and then Refresh; usually this way you can prove that the link will resolve when at the real site.

Reply:
re [][] see my comment below, I show the 2 patterns that work.  There are pros & cons (costs) of the long form and I don't know that this policy of which form to use is cast in stone.